pkg/core/native/notary.go

```diff
@@ -97,17 +100,17 @@ func newNotary() *Notary {
 	n.AddMethod(md, desc)
 
 	desc = newDescriptor("balanceOf", smartcontract.IntegerType,
-		manifest.NewParameter("addr", smartcontract.Hash160Type))
+		manifest.NewParameter("account", smartcontract.Hash160Type))
 	md = newMethodAndPrice(n.balanceOf, 1<<15, callflag.ReadStates)
 	n.AddMethod(md, desc)
 
 	desc = newDescriptor("expirationOf", smartcontract.IntegerType,
-		manifest.NewParameter("addr", smartcontract.Hash160Type))
+		manifest.NewParameter("account", smartcontract.Hash160Type))
 	md = newMethodAndPrice(n.expirationOf, 1<<15, callflag.ReadStates)
 	n.AddMethod(md, desc)
 
 	desc = newDescriptor("verify", smartcontract.BoolType,
-		manifest.NewParameter("signature", smartcontract.SignatureType))
+		manifest.NewParameter("signature", smartcontract.ByteArrayType))
```

Comment:
Meh. Bad one, but OK.

Reply:
C# node doesn't support SC types in native method wrappers out of the box, and I wasn't able to properly add this support.